internal/docstore/firedocstore/fs.go

```diff
@@ -487,6 +506,38 @@ func (c *collection) updateWrites(doc driver.Document, docName string, mods []dr
 	return []*pb.Write{w}, nil
 }
 
+func (c *collection) runActionsUnordered(ctx context.Context, actions []*driver.Action) driver.ActionListError {
+	// Split into groups the same way, but run them concurrently.
+	// TODO(jba): group without considering order.
+	groups := driver.SplitActions(actions, shouldSplit)
+	errs := make([]error, len(actions))
```

Comment:
`errs` doesn't need to be the length of the actions, see comment below.

Reply:
This is just a convenience. Only the non-nil errors are returned, see `driver.driver.NewActionListError`.